mpxplay/au_cards/sc_cmi.c

```diff
@@ -745,19 +775,33 @@ static long CMI8X38_getbufpos(struct mpxplay_audioout_info_s *aui)
  struct cmi8x38_card *card=aui->card_private_data;
  unsigned long bufpos;
 
+#if 0
```

Comment:
use #ifndef SBEMU will be better, that will keep the original code unchanged for non-SBEMU usage.

Reply:
Hold on, I will just delete that stuff.